frontend/src/components/renderData.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Can you give this a more meaningful name given it lives in components (more context),

Reply:
https://github.com/HHS/Head-Start-TTADP/pull/1598/commits/679d64a2933b598f8cd7837bb4e7aaa55ae6cc94